lib/active_metrics/connection_adapters/abstract_adapter.rb

```diff
@@ -35,6 +35,33 @@ def write(metric)
       def write_multiple(*_metrics)
         raise NotImplementedError, "must implemented by the adapter"
       end
+
+      def transform_parameters(_resources, interval_name, _start_time, _end_time, rt_rows)
```

Comment:
@Ladas I still think this belongs in `Metric::CiMixin::Processing` not in `ActiveMetrics`.  It is very specific to the `rt_rows` that we return from `Metric::CiMixin::Capture` and already calls back out to `Metric::Processing` for `process_derived_columns`

Reply:
Right, so would you rather do the parsing based on Adapter class, in the Metric::CiMixin::Processing code? (there is a different transform_parameters for the new postgre adapter)